Source/Lib/Codec/EbEncHandle.c

```diff
@@ -979,9 +979,7 @@ void EbSetThreadManagementParameters(
     // For system with a single processor group(no more than 64 logic processors all together)
     // Affinity of the thread can be set to one or more logical processors
     if (numGroups == 1) {
-        EB_U32 lps = configPtr->logicalProcessors == 0 ? numLogicProcessors:
-            configPtr->logicalProcessors < numLogicProcessors ? configPtr->logicalProcessors : numLogicProcessors;
-        groupAffinity.Mask = GetAffinityMask(lps);
```

Comment:
The logic here is for -lp setting, which is used to profile SVT scalability for cores.
May I understand why it is removed?

Reply:
I did not think through when removing this, added it back.